src/validation/schemas.js

```diff
@@ -0,0 +1,28 @@
+import Joi from '@hapi/joi';
+
+/**
+ * user schema to be used for validating user input
+ */
+const schemas = {
```

Comment:
I am guessing this schema has to take other schemas, defining what this schema is, will be of great help. Probably `signUpSchema` or `registrationSchema` and export directly instead of default export.

Example:

`
export const registerSchema = { ... }
`